dali/operators/reader/loader/coco_loader.h

```diff
@@ -65,36 +65,34 @@ inline bool HasSavePreprocessedAnnotationsDir(const OpSpec &spec) {
     (spec.HasArgument("dump_meta_files_path") && spec.GetArgument<bool>("dump_meta_files_path"));
 }
 
-struct RLEMask : public UniqueHandle<RLE, RLEMask> {
-  DALI_INHERIT_UNIQUE_HANDLE(RLE, RLEMask)
-
-  constexpr inline RLEMask() : UniqueHandle() {}
-
+struct RLEMask {
```

Comment:
Why? Shouldn't RLEMask be marked as non-movable, non-copyable if not using proper move/copy semantics?